tests/README.md

```diff
@@ -170,16 +170,17 @@ By default, stubtest emits an error if a public object is present at runtime
 but missing from the stub. However, this behaviour can be disabled using the
 `--ignore-missing-stub` option.
 
-Many third-party stubs packages in typeshed are currently incomplete, and so by
-default, `stubtest_third_party.py` runs stubtest with the
+Most third-party stubs packages in typeshed are currently complete (as far as stubtest is
```

Comment:
Github won't let me leave a suggestion, but I'd replace this paragraph with something like:

> If a distribution has `ignore_missing_stub = true` in the
`tool.stubtest` section of its `tests/METADATA.toml` file, `stubtest_third_party.py` will test that distribution with the `--ignore-missing-stub` option. This indicates that the stubs for this distribution are considered "incomplete".

Reply:
A lot less verbose, I like it. Would you keep the example?

Reply:
Nah